portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -157,4 +157,18 @@ private String getNewComment(HttpServletRequest request) throws UnsupportedEncod
     byte[] commentBytes = newComment.getBytes("UTF-8");
     return newComment;
   }
+
+  // TODO[ak47na]: create Nickname class and remove duplicate getUserNickname
```

Comment:
FYI: This change seems unrelated to the PR description of displaying sentiment scores.

Should this have been a separate PR?

It's also unused, so I'm a bit confused as to why it's included here.

Reply:
Sorry this should not have been an FYI, I am genuinely confused :)

I see this used in `createCommentEntity`, was this not compiling before and this fixes it? Can you elaborate a bit?

And try to send these fixes as their own PRs in the future, there is no such thing as a "too small PR" :)

Reply:
Yes, the `getUserNickname(String userId)` was not related to the PR. When I merged the previous PR, I accidentally deleted the function. 
I added it in this PR because the code was not compiling and I could not test it.

Thank you for the advice, I will make separete PRs for fixes from now on.